imbi/endpoints/integrations/notifications/processing.py

```diff
@@ -409,4 +404,8 @@ async def _gather_updates(self, project: ProjectInfo,
                 value = rule.pattern.resolve(body, unspecified)
                 if value is not unspecified:
                     updates[rule.fact_type_id] = value
+            else:
+                self.logger.debug('fact_type_id %s valid for %s (%r)',
```

Comment:
Is this meant to be "_in_valid"?

Reply:
yes 🤦

Reply:
fixed in https://github.com/AWeber-Imbi/imbi-api/pull/74/commits/a4d4cebd11fcb8e1b15d03979c641114a0f5f04e#diff-302a77f5b45a57f6df8eafd57dc3885345858bfbed4b8e5731b12de98e3eb9c9R408-R411